From 1b563ec1fef7b8a2740e1ee74b9f025c417f7e23 Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Wed, 22 Mar 2017 21:30:31 -0400
Subject: [PATCH rdma-core] fix udma_to_device_barrier on aarch64

The definition for udma_to_device_barrier on aarch64 is missing. Looks
like an oversight in the commit that removed the old barrier macros and
added this one.

Fixes: 85a5529dead5 ("Remove the old barrier macros")
CC: Jason Gunthorpe <jgunthorpe@obsidianresearch.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>
---
 util/udma_barrier.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/udma_barrier.h b/util/udma_barrier.h
index 86578af5..9932a687 100644
--- a/util/udma_barrier.h
+++ b/util/udma_barrier.h
@@ -93,7 +93,7 @@
 #elif defined(__sparc_v9__)
 #define udma_to_device_barrier() asm volatile("membar #StoreStore" ::: "memory")
 #elif defined(__aarch64__)
-#define wmb() asm volatile("dsb st" ::: "memory");
+#define udma_to_device_barrier() asm volatile("dsb st" ::: "memory");
 #elif defined(__sparc__) || defined(__s390x__)
 #define udma_to_device_barrier() asm volatile("" ::: "memory")
 #else
-- 
2.11.0

